x-pack/packages/security-solution/navigation/src/landing_links/landing_links_images_cards.tsx

```diff
@@ -18,83 +30,135 @@ export interface LandingLinksImagesProps {
-                    <EuiTitle size="xs" css={styles.title}>
-                      <h2>{title}</h2>
-                    </EuiTitle>
-                    {isBeta && <BetaBadge text={betaOptions?.text} />}
-                  </div>
-                }
-                titleElement="span"
-                description={<span css={styles.description}>{description}</span>}
-              />
-            </EuiFlexItem>
-          );
-        })}
-      </EuiFlexGroup>
+              <EuiIcon type="logoSecurity" />
+              <EuiTitle size="xxs">
+                <h2>{i18n.LANDING_LINKS_ACCORDION_HEADER}</h2>
+              </EuiTitle>
+            </EuiFlexGroup>
+          }
+        >
+          <EuiFlexGrid css={styles.cardsContainer} columns={3}>
+            {items.map((item) => {
+              const linkProps = getKibanaLinkProps({ item, urlState, onLinkClick });
+              const { id, landingImage, title, description, isBeta, betaOptions } = item;
+              return (
+                <EuiFlexItem key={id} data-test-subj="LandingImageCard-item" grow={false}>
+                  <EuiPanelWithLink {...linkProps} hasBorder paddingSize="xs" css={styles.card}>
+                    <EuiFlexGroup
+                      gutterSize="s"
+                      direction="row"
+                      wrap
+                      justifyContent="flexStart"
+                      alignItems="flexStart"
+                      css={styles.cardWrapper}
+                    >
+                      {landingImage && (
+                        <EuiPanel
+                          paddingSize="none"
+                          hasShadow={false}
+                          hasBorder
+                          borderRadius="m"
+                          css={styles.imageContainer}
+                        >
+                          <EuiImage
+                            data-test-subj="LandingImageCard-image"
+                            role="presentation"
+                            alt={title}
+                            src={landingImage}
+                            css={styles.image}
+                          />
+                        </EuiPanel>
+                      )}
+                      <EuiFlexGroup
+                        css={styles.cardInfoContainer}
+                        gutterSize="s"
+                        direction="column"
+                      >
+                        <div css={styles.titleContainer}>
+                          <EuiTitle size="xxxs" css={styles.title}>
+                            <h3>{title}</h3>
+                          </EuiTitle>
+                          {isBeta && <BetaBadge text={betaOptions?.text} />}
+                        </div>
+                        <span css={styles.description}>{description}</span>
+                      </EuiFlexGroup>
+                    </EuiFlexGroup>
+                  </EuiPanelWithLink>
+                </EuiFlexItem>
```

Comment:
I think this component could be encapsulated in a `<LandingImageLinkCard />` component but not sure where to place it @angorayc